src/MagicalClass.java

```diff
@@ -0,0 +1,89 @@
+import java.util.ArrayList;
+
+public abstract class MagicalClass extends PlayableCharacter {
+
+ 	// Attributes
+	private final int maxMana = 100;
```

Comment:
Shouldn't maxMana be static to maybe?

Reply:
I don't know if it's okay to defined it as static attribute. We need to discuss this with @ducampcarlos

Reply:
It could be static if it's going to be used by every Magical Class. But also, if you really need the access to that attribute, you could use the getter of it. If anyone has another opinion, you could say it, I think it could be but is not obligatory.

Reply:
Ok! I will change it to static final attribute!